### PR TITLE
[HttpKernel] do not mock removed getPublicDir() method

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Tests/KernelTest.php [hunk 1]
```diff
@@ -626,7 +626,7 @@ protected function getBundle($dir = null, $parent = null, $className = null, $bu
     {
         $bundle = $this
             ->getMockBuilder('Symfony\Component\HttpKernel\Bundle\BundleInterface')
-            ->setMethods(['getPath', 'getPublicDir', 'getParent', 'getName'])
+            ->setMethods(['getPath', 'getParent', 'getName'])
```
Comment (stof) [inline src/Symfony/Component/HttpKernel/Tests/KernelTest.php]: isn't `getParent` removed as well ?

Comment (xabbuh) [reply src/Symfony/Component/HttpKernel/Tests/KernelTest.php]: It's only deprecated and will be removed in 5.0, right?

Comment (xabbuh) [reply src/Symfony/Component/HttpKernel/Tests/KernelTest.php]: `getPublicDir()` was planned to be introduced with #31975 in 4.4 and was removed again in #32845

Comment (stof) [reply src/Symfony/Component/HttpKernel/Tests/KernelTest.php]: deprecated in 3.4 and removed in 4.0

**Review by nicolas-grekas — APPROVED** [review]

**Review by yceruto — APPROVED** [review]

**Comment by yceruto** [comment]
Thank you @xabbuh.

**Pull request merged**